scripts/update.sh

```diff
@@ -17,3 +24,9 @@ COPYRIGHT_ESCAPED='-- Copyright (c) 2024 Digital Asset (Switzerland) GmbH and\/o
 '
 
 find daml -type f -name \*.daml -exec bash -c 'grep -q -e "-- Copyright" "$0" || sed -i "1s/^/$1/" $0' '{}' "$COPYRIGHT_ESCAPED" \;
+
+for pkg in $pkgs; do
```

Comment:
this seems almost identical to the loop above, I'd suggest we factor it out into a function